packages/tailwindcss-config-analyzer/src/sort-config.ts

```diff
@@ -63,6 +76,15 @@ export function sortConfigFromSpec(
 				classes: [...new Set(classes)], // remove duplicates
 			};
 		});
+	return utilities;
+}
+
+function buildConfigVariants(spec: TailwindSpec): Array<Variant> {
+	const variants: Array<Variant> = [...spec.variants]
+		.sort((variantA, variantB) => {
```

Comment:
nit: do implicit return, and name a and b for consistent naming like in the utility code above